pkg/download/protocol.go

```diff
@@ -143,7 +148,9 @@ func (p *protocol) list(op errors.Op, mod string) (*listResp, error) {
 }
 
 func (p *protocol) Info(ctx context.Context, mod, ver string) ([]byte, error) {
-	const op errors.Op = "protocol.Info"
+	const op errors.Op = "download.protocol.Info"
```

Comment:
can we change these back to `protocol.Info` as we discussed in the other pr? as well as the other ones. Thanks!